stripper/ze_bathroom_v2_5.cfg

```diff
@@ -151,21 +161,6 @@ modify:
 	}
 }
 
-;prevent toilet paper getting stuck by weapons?
-modify:
-{
-	match:
-	{
-		"classname" "func_door_rotating"
-		"targetname" "police_door"
-	}
-	replace:
-	{
-		"forceclosed" "1"
-		"ignoredebris" "1"
-	}
-}
```

Comment:
Whether or not this change works, it still makes logical sense to have these keyvalues set on the toilet paper. Additionally, while the new weapon cleaner might make this less prevalent, it's definitely still possible (max of 10 world weapons with a 10 sec lifetime each). I'm also still unsure if we're even keeping it, as the round start lag issue has yet to be fixed.

Feel free to change the comment though, if you want it to be more accurate.